registry.go

```diff
@@ -1,13 +1,6 @@
 package distribution
 
-import (
-	"io"
-	"time"
-
-	"github.com/docker/distribution/digest"
-	"github.com/docker/distribution/manifest"
-	"golang.org/x/net/context"
-)
+import "golang.org/x/net/context"
```

Comment:
@jlhawn Please see docker/distribution#156 for a discussion of this issue. No one responded.